content/en/docs/setup/pick-right-solution.md

```diff
@@ -34,6 +34,8 @@ a Kubernetes cluster from scratch.
 
 * [Minikube](/docs/setup/minikube/) is a method for creating a local, single-node Kubernetes cluster for development and testing. Setup is completely automated and doesn't require a cloud provider account.
 
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) Docker Desktop is an easy-to-install application for your Mac or Windows environment that enables you to start coding and containerizing in minutes on a single-node Kubernetes cluster.
+
```

Comment:
Interesting use of the verb, containerizing 😄 . How is it different from Docker Enterprise, listed below? If it is different, should the desktop version be listed before the Enterprise version?

Reply:
This section is "Local Machine Solutions" which I assumed is for single host situations like laptops and desktops, which is what Docker Desktop would be targeted for. Docker Enterprise is installed on multiple servers for "Turnkey Cloud Solutions" or "On-premises Solutions" so I listed Desktop in this section only. Docker Enterprise would never be hosted on a single machine since its a scale-out container platform built with Kubernetes.

Reply:
@kbhawkey did the above answer your question or did I misinterpret the sections headings?

Reply:
Yes, thanks. I looked at the preview and `Docker Desktop` is duplicated. Would you remove the second copy?
Thanks.